ua-snap/ardac-explorer
# Adds basic umami events for clicking items and tagbar items

Closes #144 

Testing:
- Comment out line 66 in `nuxt.config.ts` so that you can trigger Umami events
- Open Umami, go to ARDAC, click on the Events tab.
- Click on an item that is rendered as a Brief (list of items on left side of home page), check that the event is tracked
- Click on an item that is rendered as a Text Item (some in the middle), check that it's tracked
- Click on an item that is rendered as a Picture Item (main story), check that it's tracked
- Click on an item in the Tagbar, check that it is tracked!
